code/modules/mob/living/living.dm

```diff
@@ -755,6 +755,8 @@
 		else
 			if(has_gravity >= GRAVITY_DAMAGE_TRESHOLD)
 				throw_alert("gravity", /obj/screen/alert/veryhighgravity)
+			else if(has_gravity < 1)
+				throw_alert("gravity", /obj/screen/alert/lowgravity)
```

Comment:
This is nonsense. There are no cases in which this happens because nobody was stupid enough to not use integers for gravity (at least not YET).

Reply:
So, if we want implement low gravity, we need to change scale? 

Now standart gravity is 1G, change it to 10 m/s2 ?

I test some low gravity and see that it shown like hight gravity, so, i make this change.